src/frontend/js/components/CourseProductCourseRuns/EnrolledCourseRun.tsx

```diff
@@ -70,6 +70,7 @@ const EnrolledCourseRun = ({ enrollment }: Props) => {
             {formatDate(enrollment.course_run.end)}
           </em>
         </div>
+        {states.error && <div className="course-runs-list__errors-feedback ">{states.error}</div>}
```

Comment:
A `<p>` makes more sens here.
```suggestion
        {states.error && <p className="course-runs-list__errors-feedback ">{states.error}</p>}
```

Reply:
agreed